packages/web/src/routes/docs/contributors/review/+page.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Reviewing Pull Requests
+---
+
+There are a lot of individual components and artifacts that make up Harper.
+How a patch gets reviewed depends significantly on which component or artifact it affects.
+
+If a patch only affects a grammar rule or otherwise something part of `harper-core` (often it is labeled as such in GitHub), you can build and test the change with **any** of Harper's frontends.
+
+## Using GitHub Actions Artifacts
+
+We run builds for a variety of platforms whenever a Pull Request is pushed to.
+You can use these to review changes to various aspects of Harper, including `harper-ls`, `harper-cli`, and the Visual Studio Code plugin.
+
+![How to download the Windows Visual Studio Code plugin from the GitHub Actions run.](/images/download_artifact.gif)
+
+## Using Cargo
+
+If you only have [Cargo](https://doc.rust-lang.org/cargo/) installed, you can compile either `harper-ls` (the language server) or `harper-cli` (our command-line debug tool) directly from the branch in GitHub:
+
+```bash
+cargo install --git https://github.com/automattic/harper --branch <branch-name> <binary-artifact>
+```
+
+For example, for [PR #445](https://github.com/Automattic/harper/pull/455), we can install the patched version of the `harper-cli` debug tool with the following command:
+
+```bash
+cargo install --git https://github.com/automattic/harper --branch somewhat-something harper-cli
+```
```

Comment:
There is something missing here, somewhere-somewhat is the name of the branch in 455.

If you don't explain it, somewhat-somewhere sounds like a meta syntaxic words like "foo-bar"

I'm reporting it, because you lost me.
I had to read carefully 5 times before understanding

Reply:
I think I'm even more lost here than you are \-: